task-manager.dart

```diff
@@ -45,14 +45,27 @@ class Task {
 }
 
 class TaskManager {
+
+  List<Task> _tasks = [];

```

Comment:
FYI:
You might want to add error handling in case the _tasks list is null or empty.